app.js

```diff
@@ -40,45 +40,45 @@ const speakers = [
       'Pichai Sundararajan, better known as Sundar Pichai, is an Indian-American business executive. He is the chief executive officer of Alphabet Inc. and its subsidiary Google. Born in Madras, India, Pichai earned his degree from IIT Kharagpur in metallurgical engineering.',
     photo: 'images/SundarPichai.png',
   },
-]
+];
 
```

Comment:
- [ ] Kindly add some spacing in-between the 2 columns of speakers

Yours | Original
--- | ---
![screenshot](https://user-images.githubusercontent.com/16058470/143688129-c81307a3-bc15-4ea5-8ad0-f11c7c3f51f0.png) | ![image](https://user-images.githubusercontent.com/16058470/143688150-9f3624eb-ec81-4999-8c49-26ecd44e43cb.png)

Reply:
spacing has been added in-between the 2 columns of speakers.

Reply:
- [ ] The spacing has still not been added

![image](https://user-images.githubusercontent.com/16058470/143723973-581edd7c-999d-4453-a47e-c97f7f76b33d.png)